lib/Controller/DelegationController.php

```diff
@@ -22,27 +22,47 @@
 
 namespace OCA\GroupFolders\Controller;
 
+use OCA\GroupFolders\AppInfo\Application;
+use OCA\GroupFolders\Service\DelegationService;
+use OCP\IConfig;
+use OCP\AppFramework\Http;
 use OCP\AppFramework\Http\DataResponse;
 use OCP\AppFramework\OCSController;
 use OCP\IGroupManager;
 use OCP\IRequest;
+use OCA\Settings\Service\AuthorizedGroupService;
 
 class DelegationController extends OCSController {
```

Comment:
I wonder if it's not interesting to define these methods in the API section from the README.md ?